src/main/java/org/apache/sling/distribution/journal/impl/subscriber/DistributionSubscriber.java

```diff
@@ -125,6 +129,15 @@ public class DistributionSubscriber {
     @Reference
     private SubscriberReadyStore subscriberReadyStore;
 
+    @Reference(name = "packageHandler")
+    PackageHandler packageHandler;
```

Comment:
shouldn't this be the `PackageHandlerFactory`? It needs a default filter defined `(name=default)`